scaffolds/nextjs-magic-auth/template/.env.example

```diff
@@ -1,5 +1,5 @@
 # Publishable API Key found in the Magic Dashboard
-NEXT_PUBLIC_MAGIC_API_KEY=
+NEXT_PUBLIC_MAGIC_API_KEY=<%= publishableApiKey %>
 
```

Comment:
Added templates so when the example file gets renamed, the app will have needed env vars